tests/Tests/400_ods_dashboard/400_ods_dashboard_auth.robot

```diff
@@ -0,0 +1,27 @@
+*** Settings ***
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library          DebugLibrary
+
+#Suite Setup      Begin Web Test
+Suite Setup      Accelerated Setup Suite
+Suite Teardown   End Web Test
+
+
+*** Keywords ***
+Accelerated Setup Suite
+  Set Library Search Order  SeleniumLibrary
+  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+
+*** Variables ***
+
+*** Test Cases ***
+
+Open RHODS Dashboard
+  [Tags]  Start Sequence
```

Comment:
what's this tag for?

Reply:
We've identified dashboard logon to be problematic at scale. We need to be able to test just the logon itself.